plugin.json

```diff
@@ -5,68 +5,52 @@
     "main": "index.js",
     "config": [
         {
-            "key": "hubspotApiKey",
-            "hint": "Can be acquired under Profile Preferences -> Integrations",
-            "name": "Hubspot API Key",
+            "key": "actionIdToName",
+            "hint": "Comma-separated map from action ids to names. No spaces between entries.",
+            "name": "Action IDs to Conversion Names (map)",
             "type": "string",
-            "default": "",
+            "default": "11036:Sign up - cloud,11037:Sign up - self-hosted free,11038:Sign up - self-hosted paid",
             "required": true
         },
         {
-            "key": "triggeringEvents",
-            "hint": "A comma-separated list of PostHog events you want to trigger Contact creation in HubSpot. By default, we recommend using the $identify event.",
-            "name": "Triggering events",
+            "key": "defaultStartTime",
+            "hint": "Point in time from which to start fetching data. Should be ISO format for best results.",
```

Comment:
I think this _needs_ to be in ISO for it to work, so let's use stronger language here?

Reply:
This can be any format accepted by the `Date()` constructor, so more specifically will include RFC 2822 timestamps. Will update!